google/cloud/aiplatform/models.py

```diff
@@ -198,6 +198,7 @@ def network(self) -> Optional[str]:
     def create(
         cls,
         display_name: Optional[str] = None,
+        endpoint_id: Optional[str] = None,
```

Comment:
Please add this field to the end of the signature to avoid breaking changes on order of arguments(not necessary on private methods).

Reply:
Thanks for your review! Moved the field to the end(also on the private method): https://github.com/googleapis/python-aiplatform/pull/1168/commits/14fa07104b3c90cf2b06cdc4b895e9166cf4ef7c